dev.sh

```diff
@@ -4,6 +4,13 @@ VENV=./venv
 
 python -m virtualenv $VENV --always-copy
 . $VENV/bin/activate
+pip_version=$(pip -V | cut -d' ' -f 2 | cut -d'.' -f 1)
+if [ "$pip_version" -lt 6 ];
```

Comment:
3.x is enough, see my comment in #972. Nonetheless, let's just upgrade to the latest thing available.